readthedocs/projects/querysets.py

```diff
@@ -113,7 +116,11 @@ def max_concurrent_builds(self, project):
         return (
             project.max_concurrent_builds or
             max_concurrent_organization or
-            settings.RTD_MAX_CONCURRENT_BUILDS
+            PlanFeature.objects.get_feature_value(
+                project,
+                type=PlanFeature.TYPE_CONCURRENT_BUILDS,
+                default=settings.RTD_MAX_CONCURRENT_BUILDS,
+            )
```

Comment:
we had an override just to check for the value from the plan on .com.

Reply:
It seems my suggestion is already applied, right?